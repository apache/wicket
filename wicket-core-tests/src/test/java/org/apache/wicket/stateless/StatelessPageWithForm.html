<html>
<body>
<form wicket:id="form"></form>
</body>
</html>