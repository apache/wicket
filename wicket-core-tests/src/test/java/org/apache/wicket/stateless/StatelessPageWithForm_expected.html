<html>
<body>
<form wicket:id="form" id="form1" method="post" action="./org.apache.wicket.stateless.StatelessPageWithForm?-1.-form&amp;first=foo&amp;second=bar"><div id="form1_hf_0" hidden="" class="hidden-fields"></div></form>
</body>
</html>